microsoft-azure-sb/3.2/modules/ROOT/pages/azure-service-bus-connector-studio.adoc

```diff
@@ -0,0 +1,181 @@
+.. Enter the Azure *Service namespace*. +
+This is the unique name assigned to the namespace when it was created and is the host-name portion of the URL, for example:
++
+`+https://{serviceNamespace}.servicebus.windows.net/{path}+`
++
+The following screenshot shows an example of configuring SAS authentication for Azure Service Bus Connector:
++
+.SAS authentication fields
+image::azure-service-bus-sas-connection.png[To configure authentication, complete the fields on the *General* tab.]
++
+.. Enter the Azure *Shared access key name*. +
+This is the name of the SAS token configure in the Azure Service Bus namespace.
+.. Enter the Azure *Shared access key*. +
+This is the name of your SAS token.
++
+You can reference a configuration file that contains ANT-style property placeholders (recommended), or you can enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[NOTE]
+
+Azure Service Bus Connector cannot perform a connectivity test at startup using the `Test Connection` button in the global element of the connector configuration when you restrict access to your resources and you have a security policy with permissions at the resource level only. This is because the security policy targets the root level of the namespace, which might be forbidden due to the customized policy applied to the shared access key.
+
+[[configure-global-element-aad-connection]]
+=== Azure Active Directory (AAD) authentication
+
+To configure Azure Active Directory authentication for Azure Service Bus Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, select the `Active Directory Connection` in the connection dropdown, and configure the following required fields for AAD authentication:
+.. Enter the Azure *Service namespace*. +
+This is the unique name assigned to the namespace when it was created and is the host-name portion of the URL, for example:
++
+`+https://{serviceNamespace}.servicebus.windows.net/{path}+`
++
+The following screenshot shows an example of configuring AAD authentication for Azure Service Bus Connector:
++
+.AAD authentication fields
+image::azure-service-bus-aad-connection.png[To configure authentication, complete the fields on the *General* tab.]
```

Comment:
This image seems out of place. If it shows the Tenant ID, Client ID, etc., it should be after steps for entering those fields. I can't see the picture, so I can't really tell.